include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -379,10 +379,10 @@ class argument_parser
      * A list item is composed of a key (`key`) and value (`desc`)
      * and usually used for option identifier-description-pairs.
      * E.g.:
-     *-------------------------------------------
+     *===========================================
```

Comment:
```suggestion
    ```console
```
and ` ``` ` at the end, instead of the second `===========================================`